components/cdc/src/observer.rs

```diff
@@ -123,6 +124,7 @@ impl<E: KvEngine> CmdObserver<E> for CdcObserver {
                 RegionSnapshot::from_snapshot(Arc::new(engine.snapshot()), Arc::new(region));
             let mut reader = OldValueReader::new(snapshot);
             let get_old_value = move |key, old_value_cache: &mut OldValueCache| {
+                CDC_OLD_VALUE_CACHE_ACCESS.inc();
```

Comment:
Could you add two fields to OldValueCache and flush them to metrics in `Endpoint::on_timeout()`? In this way, we can avoid unnecessary atomic access.